src/components/sidebar/ToggleInverseKinematics.css

```diff
@@ -46,11 +46,16 @@
     height: 100%;
     border-radius: 100px;
     outline: 2px solid white;
+    box-sizing: border-box;
```

Comment:
This element has an outline, not a border, so changing box-sizing to border-box seems to have no effect.